### DISCUSSION TITLE
Dark code syntax highlighting in Light mode?

### DISCUSSION THREAD

**Discussion by johnchaffee** [thegeeklab/hugo-geekdoc#427] [Q&A] [answered]
Hi. Thanks for creating the Geekdoc theme. It's really great. 

I use the Light mode but would prefer that the code syntax highlighting appeared on a dark background like that in the Dark mode.

Is it possible to mix and match them? If not, can I put in a feature request? :)

Here's a mockup of what I am envisioning.

Before
![before](https://user-images.githubusercontent.com/428417/171107145-c6b3dcfc-27c9-4c7e-a336-aaf1f5c8e829.png)

After
![after](https://user-images.githubusercontent.com/428417/171107163-e20c05ef-d710-4c92-aa85-83fd2adafbb8.png)

**Comment by xoxys** [comment]
Hi, Thanks for your kind words and the feature request. In general that should be possible but will apply to all code blocks including inline code:

![image](https://user-images.githubusercontent.com/3391958/171151897-fdba3a48-3343-4008-ae59-c994877b0680.png)

**Reply by johnchaffee** [reply]
Wow! I did not expect an answer so quickly, let alone a feature implementation. Thank you!

I think that's a great option. Any chance that the inline code could be rendered in the light style while rendering the code blocks in the dark style? You knew I was going to ask that next... :)

**Reply by xoxys** [reply]
Not without a lot of effort, sorry.

**Reply by johnchaffee** [reply]
Fair enough. I don't want to be greedy. How can I get the latest changes? Will it be in the next release?

Thanks again.

**Reply by xoxys** [reply]
No problem :) Yes will be included in the next release (will tag on later today or tomorrow). You can also build it on your own from main branch if you want, see https://geekdocs.de/usage/getting-started/#option-2-clone-the-github-repository

**Reply by johnchaffee** [reply]
Thanks again for making this change. I really like it.

I was able to make the code blocks dark using the new `geekdocDarkModeCode = true` option while keeping the inline code with a light background by hacking together the following in my custom.css:

```css
/* Override inline code background color */
.gdoc-markdown p > code,
.gdoc-markdown li > code {
  background-color: #f5f6f8;
  color: rgb(95, 95, 95);
}
```

**Comment by xoxys** [comment, marked as answer]
Implemented in https://github.com/thegeeklab/hugo-geekdoc/pull/429 Please let me know what you think.
